### PR TITLE
Add calculator v1

### PR DESCRIPTION
**Pull request by dkjunior**
* Calculator initial version
* Only supports add and subtract

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    public Calculator() {
+    }
+
+    public int add(int a, int b) {
+        return a + b;
+    }
+
+
+    public int Subtract(int a, int b) {
+        return a + b;
```
Comment (arthur-123) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java:13]: Should've been '-'

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+public class Calculator {
+    public Calculator() {
+    }
+
+    public int add(int a, int b) {
+        return a + b;
+    }
+
+
+    public int Subtract(int a, int b) {
```
Comment (arthur-123) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java:12]: lowercase

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package org.firstinspires.ftc.teamcode;
+
+public class Calculator {
+    public Calculator() {
+    }
+
+    public int add(int a, int b) {
+        return a + b;
+    }
+
```
Comment (arthur-123) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java:10]: extra line

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java (hunk 1 above)
Comment (dkjunior) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java:13]: Good catch! Thank you, I'll fix it

Comment (dkjunior) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Calculator.java]: Fixed everything, PTAL

**Comment by arthur-123** [comment]
LGTM

**Pull request merged**